docker-storage-setup.sh

```diff
@@ -537,6 +537,32 @@ is_block_dev_partition() {
   return 1
 }
 
+check_block_dev_sig() {
+  local bdev=$1
+  local sig
+
+  [ ! -b "$bdev" ] && bdev="/dev/${bdev}"
+
+  if ! sig=$(wipefs -p $bdev); then
+    Fatal "Failed to check signatures on device $bdev"
+  fi
+
+  [ "$sig" == "" ] && return 0
+
+  while IFS=, read offset uuid label type; do
+    [ "$offset" == "# offset" ] && continue
+
+    # Ignore dos signatures for now. We have already checked that disk
+    # does not have partitions. But there might be leftover partition
+    # signature and so far we have not forced user to wipe it. So in an
+    # attempt to not change user visible behavior, don't force user to
+    # wipe dos signature and just make sure disk is not partitioned.
```

Comment:
It sounds like the special case isn't `dos`, it's "partition tables" which also include `gpt`, right?

That said...I think we *should* require people to understand how to use `wipefs` on their own rather than doing it specially for them just for disks with empty (but initialized) partition tables.